Proposal for the basic structure of a simulation study.md

```diff
@@ -0,0 +1,49 @@
+# Proposal for the basic structure of a simulation study
+
+### Final Structure
+
+1. *Defining aims & objectives (including hypotheses)*
+2. *Simulation procedure /Design & Data-generation mechanism*
+3. *Methods selection (including software and estimands)*
+4. *Optional: Model specification*
+5. *Selection of performance measures*
+6. *Coding & execution*
+7. *Analyzing results*
+8. *Presentation/ Reporting*
```

Comment:
I like the notion to have analyzing and reporting separate but it might be hard to disentangle as the chosen method of analysis directly implies certain types of presentation and reporting.

Reply:
I agree. I think I implictly meant to include interpretation in Step 8 and will double check on that adn make it explicit

Reply:
It was not about interpretation. Still, I think this should be seperated as it has been done in many papers (Burton et al., 2006; Morris et al., 2019; Smith & Marshall, 2010).